scripts/node_e2e_installer

```diff
@@ -30,20 +30,22 @@ install_crio() {
     # Setup log level
     echo "CONTAINER_LOG_LEVEL=debug" >>/etc/sysconfig/crio
 
+    runc_path=$(command -v runc || true)
```

Comment:
Do we wanna fail if it's not in $PATH? Or: do we wanna fallback to `/usr/local/bin/runc` in that case?

Reply:
@haircommander yeah CRI-O should find it in $PATH as well :+1: